src/shims/dir.props

```diff
@@ -25,6 +25,6 @@
   </ItemGroup>
 
   <PropertyGroup>
-  	<GenFacadesOutputPath>$(IntermediateOutputPath)facades/</GenFacadesOutputPath>
+  	<GenFacadesOutputPath>$(BaseIntermediateOutputPath)/shims/facades/</GenFacadesOutputPath>
```

Comment:
I guess that is true for the netstandard shim, the netfx ones should in theory only be built by netcoreapp vertical now. Does `$(BaseIntermediateOutputPath)/shims/facades/$(TargetGroup)/` sound good to you?